weather_app/weather/views.py

```diff
@@ -228,6 +228,46 @@ def get(self, request):
     def post(self, request):
         print(request.POST)
     
+    
+class VistaPostFormulario(View):
+    def get(self, request):
+        
+        return render(request, 'weather/vistapost.html')
+    
+    def post(self, request):
+        primernombre= request.POST.get ('primer_nombre')
+        segundonombre = request.POST.get('segundo_nombre')
+        primerapellido = request.POST.get('primer_apellido')
+        segundoapellido = request.POST.get('segundo_apellido')
+        lista = request.POST.get('tipo_documento')
+        numeroidentidad = request.POST.get('numeroIdentidad')
```

Comment:
```suggestion
        numeroidentidad = request.POST.get('numero_identidad')
```
Si cambias el nombre arriba, tambien te tocaria cambiarlo aqui porque la llave la va a tomar del name del campo